.travis.yml

```diff
@@ -4,3 +4,4 @@ script:
 jdk:
    - oraclejdk7
    - openjdk7
+sudo: false
```

Comment:
Honestly I don't remember how I figured out that this would fix the build, but it does. On the plus side, disabling sudo lets you use the Docker infrastructure, which has more capacity, and shorter queues. Since sudo is unnecessary for the build, it doesn't hurt to turn it off.